src/shared/components/palette/dynamic-list/write-dynamic-list-field.component.ts

```diff
@@ -23,8 +23,10 @@ export class WriteDynamicListFieldComponent extends AbstractFieldWriteComponent
      *
      * Reassigning value from formatted_value when value is empty
      */
-    if (!this.caseField.value && this.caseField.formatted_value && this.caseField.formatted_value.value && this.caseField.formatted_value.value.code) {
-      this.caseField.value = this.caseField.formatted_value.value.code;
+    if (!this.caseField.value) {
```

Comment:
Any particular reason for splitting `!this.caseField.value` into a separate `if` condition, or just for better readability?

Reply:
It was a lint error as the line was over 140 characters so I either wrapped one long conditional onto multiple lines or I split it the way I did. Stylistically, I preferred to split them and group the `value` and `formatted_value` checks. 🤷

Reply:
That's fair enough 🙂